dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/ESMappingUtilHelper.java

```diff
@@ -352,7 +355,7 @@ private void addMappingForFieldIfNeeded(
      * @param fieldVariableName
      * @return A map with just one element
      */
-    private Optional<Tuple2<String, JSONObject>> getMappingForField(final Field field, final String fieldVariableName)
+    private Optional<List<Tuple2<String, JSONObject>>> getMappingForField(final Field field, final String fieldVariableName)
```

Comment:
I think a simple list return type would do better ... instead of an optional list. I mean list has an empty state

Reply:
agree